src/Microsoft.TestPlatform.VsTestConsole.TranslationLayer/VsTestConsoleProcessManager.cs

```diff
@@ -106,6 +107,13 @@ public void StartProcess(ConsoleParameters consoleParameters)
 
             EqtTrace.Verbose("VsTestCommandLineWrapper: Process Start Info {0} {1}", info.FileName, info.Arguments);
 
+#if NET451
+            info.EnvironmentVariables.Clear();
```

Comment:
Should we only clear if environment variables are supplied? Might affect other clients otherwise.